docs/casestudies/adopters.md

```diff
@@ -12,13 +12,14 @@ To join this list, please follow [these instructions](https://github.com/karmada
-| Ecloud                    | https://ecloud.10086.cn/    | Use Karmada to build a steady and elastic multi-cluster platform for managing workloads | TBD                                                                           |
-| Hurricane Engines         | /                           | Use Karmada to build multi-cloud PaaS platform with multi-vendor and multi-region       | [Karmada in AIML INSTITUTE](ci123.md)                                         |
-| Trip.com                  | https://www.trip.com/       | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada            | TBD                                                                           |
-| VIPKID                    | https://www.vipkid.com/     | Use Karmada to build multi-cluster platform with heterogeneous resources and clusters   | [Building a PaaS Platform with Karmada to Run Containers --VIPKID](vipkid.md) |
-| Vivo                      | https://www.vivo.com/       | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada            | TBD                                                                           |
-| Youzan                    | https://www.youzan.com/     | Manage both Stateless and Stateful on-line workloads across multiple clusters           | TBD                                                                           |
\ No newline at end of file
+| Organization/Company Name | Website                      | Usage Scenario                                                                          | CaseStudy                                                                     |
+|---------------------------|------------------------------|-----------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|
+| Cibfintech.com            | https://www.cibfintech.com/  | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada            | TBD                                                                           |
+| DaoCloud.io               | https://www.daocloud.io/     | Build a more resilient and elastic hybrid-multi-cloud platform using Karmada            | TBD                                                                           |
+| Ecloud                    | https://ecloud.10086.cn/     | Use Karmada to build a steady and elastic multi-cluster platform for managing workloads | TBD                                                                           |
+| Huawei                    | https://www.huaweicloud.com/ | Build an enterprise-grade multi-cloud container platform with Karmada                   | TBD                                                                           |
```

Comment:
@GitHubxsy Any comments? 
I think a link to the service would be great.

Reply:
[Ubiquitous Cloud Native Service（UCS）](https://www.huaweicloud.com/product/ucs.html)   
UCS product link：https://www.huaweicloud.com/product/ucs.html

Reply:
Done